Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pwasm-std"
-version = "0.10.0"
+version = "0.11.0-beta.0"
```

Comment:
please don't update any metadata in the pr itself, it will be separate commit

Reply:
done